packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -39,13 +39,17 @@ export async function modelCard(documentId: string): Promise<void> {
 export async function configureModelFromDocument(
 	documentId: string,
 	modelId: string,
-	handlers: TaskEventHandlers
+	handlers: TaskEventHandlers,
+	workflowId?: string,
+	nodeId?: string
 ): Promise<TaskHandler | null> {
 	try {
 		const response = await API.get<TaskResponse>('/gollm/configure-model', {
 			params: {
 				'model-id': modelId,
-				'document-id': documentId
+				'document-id': documentId,
+				'workflow-id': workflowId,
+				'node-id': nodeId
```

Comment:
I wonder if it would be better to create a optional-object, like tracker/tagger, than explicitly spell out all the associated Ids, cause while this works, it feels very messy since they are optionals, and maybe a bit error prone as well.

So something like this perhaps??
```
params: {
  model-id: ...,
  document-id: ...,
  tagger: { key-value-pair }
}
```


@YohannParis @dvince2 @jryu01 thoughts?

Reply:
I wasn't sure if there was a simple way to pass an arbitrary object as a parameter value with this spring annotation magic. When I first tried it didn't allow me to do so.

Reply:
As we discussed I'll think about it and implement similar to what you've mentioned in the follow up work.